OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnDeploy.cs

```diff
@@ -57,6 +58,9 @@ public class GrantConditionOnDeployInfo : ITraitInfo
 		[Desc("Can this actor undeploy?")]
 		public readonly bool CanUndeploy = true;
 
+		[Desc("Assigned sprite body (used to play DeployAnimation).")]
+		public readonly string AssignedBodyName = "body";
```

Comment:
Although ideally we would remove the explicit dependency on `WithSpriteBody` completely, and use a notification instead.